apps/io-profile/src/main.ts

```diff
@@ -0,0 +1,242 @@
+import { geoLocationServiceClient } from "./utils/geo-location";
+import { getMagicCodeActivityHandler } from "./functions/get-magic-code-activity";
+import { getMagicLinkServiceClient } from "./utils/magic-code";
+import { GetServicesPreferencesActivityHandler } from "./functions/get-services-preferences-activity";
+import { getNoticeLoginEmailOrchestratorHandler } from "./functions/notice-login-email-orchestrator";
+import { getSendLoginEmailActivityHandler } from "./functions/send-templated-login-email-activity";
+import { getSendValidationEmailActivityHandler } from "./functions/send-templated-validation-email-activity";
+import getSendWelcomeMessagesActivityFunction from "./functions/send-welcome-messages-activity";
+import { updateSubscriptionFeed } from "./functions/update-subscriptions-feed-activity";
+import { getUpsertedProfileOrchestratorHandler } from "./functions/upserted-profile-orchestrator";
+import { getCreateValidationTokenActivityHandler } from "./functions/create-validation-token-activity";
+import { randomBytes, toHash } from "./utils/crypto";
+import { HTML_TO_TEXT_OPTIONS } from "./utils/email";
+
+// HTTP external requests timeout in milliseconds
+const REQUEST_TIMEOUT_MS = 5000;
+const config = getConfigOrThrow();
+
+const telemetryClient = initTelemetryClient();
+
+const timeoutFetch = getTimeoutFetch(REQUEST_TIMEOUT_MS as Millisecond);
+
+const cosmosDbName = config.COSMOSDB_NAME;
+const cosmosDbConnectionString = config.COSMOSDB_CONNECTION_STRING;
+
+const cosmosdbClient = new CosmosClient(cosmosDbConnectionString);
+
+const cosmosdbInstance = cosmosdbClient.database(cosmosDbName);
+
+const userDataProcessingModel = new UserDataProcessingModel(
+  cosmosdbInstance.container(USER_DATA_PROCESSING_COLLECTION_NAME),
+);
+
+const profileModel = new ProfileModel(
+  cosmosdbInstance.container(PROFILE_COLLECTION_NAME),
+);
+const serviceModel = new ServiceModel(
+  cosmosdbInstance.container(SERVICE_COLLECTION_NAME),
+);
+const servicePreferencesModel = new ServicesPreferencesModel(
+  cosmosdbInstance.container(SERVICE_PREFERENCES_COLLECTION_NAME),
+  SERVICE_PREFERENCES_COLLECTION_NAME,
+);
+
+const activationModel = new ActivationModel(
+  cosmosdbInstance.container(ACTIVATION_COLLECTION_NAME),
+);
+
+const profileEmailReader = new DataTableProfileEmailsRepository(
+  getProfileEmailTableClient(
+    config.PROFILE_EMAIL_STORAGE_CONNECTION_STRING,
+    config.PROFILE_EMAIL_STORAGE_TABLE_NAME,
+  ),
+);
+
+const TOKEN_INVALID_AFTER_MS = (1000 * 60 * 60 * 24 * 30) as Millisecond; // 30 days
+
+const tableService = createTableService(config.QueueStorageConnection);
+
+// When the function starts, attempt to create the table if it does not exist
+// Note that we cannot log anything just yet since we don't have a Context
+tableService.createTableIfNotExists(VALIDATION_TOKEN_TABLE_NAME, () => 0);
+
+tableService.createTableIfNotExists(config.SUBSCRIPTIONS_FEED_TABLE, () => 0);
```

Comment:
I think we can safely remove this code

Reply:
done in https://github.com/pagopa/io-auth-n-identity-domain/pull/256/commits/57c89d6f4be2f7a52ce8defaa8e922d8f7c957f5